fuzz/fuzz_targets/groups_message.rs

```diff
@@ -200,19 +200,19 @@ impl StateMachine {
     fn suggest_valid(&self, try_options: &[Options], rng: &Rng) -> Operation {
         let mut options = Vec::new();
 
-        if try_options.contains(&Options::Add) {
-            if let Some(added) = self.values.random_member(rng) {
-                options.push(Operation::Add {
-                    added,
-                    members_in_welcome: self.values.active_members.clone(),
-                });
-            }
+        if try_options.contains(&Options::Add)
+            && let Some(added) = self.values.random_member(rng)
+        {
+            options.push(Operation::Add {
+                added,
+                members_in_welcome: self.values.active_members.clone(),
+            });
         }
 
-        if try_options.contains(&Options::Remove) {
-            if let Some(removed) = self.values.random_active_member(rng) {
-                options.push(Operation::Remove { removed });
-            }
+        if try_options.contains(&Options::Remove)
+            && let Some(removed) = self.values.random_active_member(rng)
+        {
+            options.push(Operation::Remove { removed });
```

Comment:
Oh, interesting. What is this change in the `p2panda-encryption` fuzz test doing?

Reply:
Ah, this is a clippy suggested change, I can revert it from this PR.